src/applications/vaos/appointment-list/components/AppointmentDateTime.jsx

```diff
@@ -1,25 +1,30 @@
-import React from 'react';
+import { formatInTimeZone } from 'date-fns-tz';
 import PropTypes from 'prop-types';
-import moment from 'moment';
+import React from 'react';
 import { getAppointmentTimezone } from '../../services/appointment';
 
-export function AppointmentDate({ date, format = 'dddd, MMMM D, YYYY' }) {
-  return <> {moment.parseZone(date).format(format)} </>;
+export function AppointmentDate({
+  date,
+  timezone,
+  format = 'EEEE, MMMM d, yyyy',
+}) {
+  return <> {formatInTimeZone(date, timezone, format)} </>;
 }
 AppointmentDate.propTypes = {
   date: PropTypes.object,
   format: PropTypes.string,
+  timezone: PropTypes.string,
 };
 
-export function AppointmentTime({ appointment, format = 'h:mm a' }) {
+export function AppointmentTime({ appointment, timezone, format = 'h:mm a' }) {
```

Comment:
Hmm I know we don't want to update files outside of appointment-list as much as possible but there is a referrals page that use this shared component [here](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/referral-appointments/EpsAppointmentDetailsPage.jsx#L130). It would display an incorrect time without timezone being passed in though right? Also we might need to talk to them about how they are formatting their date since they don't use the shared component nor do they account for timezones it seems.

Reply:
Good point.